lib/Sema/DerivedConformanceCodable.cpp

```diff
@@ -1669,7 +1669,7 @@ deriveBodyDecodable_enum_init(AbstractFunctionDecl *initDecl, void *) {
           auto *nestedContainerDecl = createKeyedContainer(
               C, funcDC, C.getKeyedDecodingContainerDecl(),
               caseCodingKeys->getDeclaredInterfaceType(),
-              VarDecl::Introducer::Var, C.Id_nestedContainer);
+              VarDecl::Introducer::Let, C.Id_nestedContainer);
```

Comment:
The generated code would otherwise produce a warning:
`Variable 'nestedContainer' was never mutated; consider changing to 'let' constant`

The example in the doc comment of this method already said `let nestedContainer`. So this looks like it was a small oversight (which didn't really affect anything so far, I imagine).